src/legacy/ui/public/vis/request_handlers/request_handlers.d.ts

```diff
@@ -42,7 +42,7 @@ export interface RequestHandlerParams {
   visParams?: any;
 }
 
-export type RequestHandler = <T>(params: RequestHandlerParams) => T;
+export type RequestHandler<T = unknown> = (params: RequestHandlerParams) => T;
```

Comment:
moving the type parameter to the outer type to be able to specify it in courier-specific type. Inferred types in other usages should work like before.